web/src/main/java/org/cbioportal/web/SessionServiceController.java

```diff
@@ -297,10 +307,9 @@ public void updateUsersInVirtualStudy(@PathVariable SessionType type, @PathVaria
     public ResponseEntity<List<VirtualStudy>> fetchUserGroups(
             @Size(min = 1, max = PagingConstants.MAX_PAGE_SIZE) @RequestBody List<String> studyIds,
             HttpServletResponse response) throws IOException {
+        if ((isSessionServiceEnabled() && isAuthorized()) || isNoAuthSessionServiceActive()) {
+            String username = isNoAuthSessionServiceActive() ? "anonymous" : userName();
```

Comment:
default username is `anonymous` for `social_auth_google` and `social_auth_microsoft` authenticate types. Using this for `noauthsessionservice` would fetch all groups created with the those authenticate modes.

Reply:
@kalletlak The goal here is to have session service to work when locally testing w/o authentication getting in the way. I have no other concerns and, due to limited/no understanding of session service, I cannot see beyond the current proposa.. Do you have a recommendation here?